### ISSUE TITLE
Unable to build ox s package on ox x

### ISSUE THREAD

**Issue by Allasso** [nwjs-community/nw-builder#511]
I have a new setup of nwjs on OS X 10.13.6.  I have installed nw-builder (both globally and locally in attempt to try to solve my issue):

    npm install nw-builder --save-dev
    sudo npm install nw-builder -g

(IIRC I needed to sudo to install globally without error, and also IIRC I tried uninstalling and re-installing nw-builder)

After `npm install`, I get error running:

    npm run prod
    ...
    Create release folder in /Users/kevin/__projects/NWJS/zcbenz0/dist/NWJSTemplate/osx64
    [ { Error: EINVAL: invalid argument, readlink '/Users/kevin/__projects/NWJS/zcbenz0/dist/NWJSTemplate/osx64/NWJSTemplate.app/Contents/Resources/app.nw/node_modules/nw/nwjs/nwjs.app/Contents/Versions/66.0.3359.181/nwjs Framework.framework/Helpers'

Full output and log (pastebin and attached):

[paste_nw-builder_error.txt](https://github.com/nwjs-community/nw-builder/files/2894491/paste_nw-builder_error.txt)
https://pastebin.com/rmngDy8F

I tried running as sudo as well to no avail.

I have uploaded copies of the package (pre-installed and post-installed versions) I am trying to build to my server:

http://kevinallasso.org/nwjs/zcbenz0_pre_install.zip
http://kevinallasso.org/nwjs/zcbenz0_post_install.zip

I have been successfully using and building nwjs apps on my laptop but ran into problems when using for the first time on new desktop computer.

**Comment by Blatman** [comment]
No expert in all this but in my hacking around with nw-builder I found that if a copy of nwjs exists in the source folder being packaged then this sort of error can occur. nw-builder will use a specified version of nwjs and will not expect to find a copy within the node_modules (in resources/app.nw ) of the source being packaged.
Looking at the errors above it looks like a copy of nwjs is residing within the app.nw folder which is itself being packaged into a copy of nwjs.

**Comment by Allasso** [comment]
I've found that when I `install` and `run prod` a 2-level package (source files in a sub-directory "src" under the root directory with its own package.json) everything works fine.  I don't quite have my head around why this works yet, but I suspect this is related to @Blatman's comments.

Weird though, I could swear I've built single-level packages before, but maybe I don't have package.json quite right this time.

Thanks very much for your input, Blatman.
